File: final/final.py
import random

def avg(nums):
	return sum(nums) / 100

def greaterCount(nums, avg):
	count = 0
	for i in nums:
		if (i > avg):
			count+=1
	return count

def getRandomNum(min, max):
	return random.randint(min, max)

nums = []
for i in range(100):
	nums += [random.randint(-11, 83)]

print(nums)

print("Average: ", avg(nums), " Count: ", greaterCount(nums, avg(nums)))

File: final/final.pl
use strict;
use warnings;
use diagnostics;

use 5.32.0;

sub avg {
	my @arr = @{$_[0]};
	my $sum = 0;
	for my $i (@arr) {
		$sum += $i;
	}
	return $sum / 100;
}

sub greaterCount {
	my ($avg, @arr) = @_;
	my $count = 0;
	for my $i (@arr) {
		if ($i > $avg) {
			$count++;
		}
	}
	return $count;
}

sub main {
	my @nums = (1..100);

	for(my $i = 0; $i < 100; $i++) {
		$nums[$i] = int(rand 95) - 11;
	}

	print("@nums\n");

	my $avg = avg(\@nums);
	my $count = greaterCount($avg, @nums);
	print("Average: ", $avg, " Count: ", $count, "\n");

}

main();